### ISSUE TITLE
Is this package abandoned?

### ISSUE THREAD

**Issue by Johnlon** [pborman/getopt#21]
I notice there has been no activity for a couple of years and that Issues persist for years without a response from the maintainer.

Is this package abandoned?

**Comment by pborman** [comment]
No, it is not abandoned, I just don’t look at it very often now that I am retired :-)

I am adding some updates to it as well as the related options package.
